frontend/integration-tests/protractor.conf.ts

```diff
@@ -59,7 +59,6 @@ const testSuites = {
     'tests/crud.scenario.ts',
     'tests/secrets.scenario.ts',
     'tests/filter.scenario.ts',
-    'tests/modal-annotations.scenario.ts',
```

Comment:
We should leave the scenario and use `xdescribe` so we see they're disabled in the log and JUnit report.

Reply:
The downside to that is `beforeAll` and `afterAll` will still run...

Reply:
> The downside to that is `beforeAll` and `afterAll` will still run...

i should say, `base.scenario.ts` including `Create a test namespace` will still run.

Reply:
I think that only runs once for the entire e2e suite?

Reply:
it runs for each suite.  but it's fine.  dave's gonna remove these soon anyhow.

Reply:
yes, and I just checked a few annotation failures, they seem to happen during the actual test, not in before hooks:
```